lua/wire/client/text_editor/wire_expression2_editor.lua

```diff
@@ -1887,19 +1914,33 @@ function Editor:Setup(nTitle, nLocation, nEditorType)
 	local useValidator = nEditorType ~= nil
 	local useSoundBrowser = nEditorType == "SPU" or nEditorType == "E2"
 	local useDebugger = nEditorType == "CPU"
+	local useColorMixer = nEditorType == "E2" or nEditorType == "GPU"
 
 	if not useValidator then
 		self.C.Val:SetVisible(false)
 	end
 
-	if useSoundBrowser then -- Add "Sound Browser" button
+	-- PickColorFind
+	if not useSoundBrowser then self.C.SoundBrw:SetVisible(false) end
+	if not useColorMixer then self.C.PickColor:SetVisible(false) end
+
+	--[[if useColorMixer then
+		local CMixBrowser = vgui.Create("Button", self.C.Menu)
+		CMixBrowser:SetSize(60, 20)
+		CMixBrowser:Dock(RIGHT)
+		CMixBrowser:SetText("Pick Color")
+		CMixBrowser.DoClick = function() RunConsoleCommand("wire_pickcolor_browser_open") end
+		self.C.CMixBrowser = CMixBrowser
+	end]]--
+
+	--[[if useSoundBrowser then -- Add "Sound Browser" button
```

Comment:
Did you forget to move this if statement when you moved the sound browser button? Pretty sure the sound browser button shouldn't be visible outside of E2, but with your new button it looks like it's always visible.